feugy/melodie
# feat: includes album artists in artist list, display in tracks tables

### What's in there?

Album artists come in addition to track artists. They are very handy to group tracks in composition albums.
They are also used as "virtual" artists, such as Audio machine who is a consortium (or a studio).
This PR improves their support to album artist can appear in the artist list page, and in search results

Are included here:

- [x] feat(core): enriches tracks with a ref on their `albumartist`
- [x] feat(ui): displays multiple artists in tracks tables and details
- [x] doc: updates release instructions
- [x] refactor(ui): replaces Storybook with Atelier!
- [x] refactor(site): replaces Storybook with Atelier FTW!
- [x] chore(ui): solves styles.svelte compiler warning
- [x] chore(site): solves styles.svelte compiler warning
- [x] chore: properly supports svelte in eslint configuration

### How to test?

Assuming you have several albums of the same `albumartist` but different `artist`:
- open Mélodie
- in settings, remove all tracked folders
- restart Mélodie twice (this will reset the track list)
- in settings, adds your folders back
- in the artist list, your `albumartist` should appear
- search for your `albumartist` name: it should appear in results
- open any album of that `albumartist`: it should appear in the artist column of the tracks table, together with the track `artist`

### Notes to reviewers

FINALLY!
Storybook is out of the picture, opening the gates for a postcss upgrade.
Atelier is nowhere as complete as Storybook, but it's lightning fast, supports Svelte much better, and provides all the feature I need!
